src/app/utils.test.ts

```diff
@@ -344,3 +346,148 @@ describe("shortenWalletAddress", () => {
     expect(result).toBe(expectedShortened);
   });
 });
+
+describe("searchPairs", () => {
+  const pairsList: any = [
+    {
+      name: "DEXTR/XRD",
+      token1: {
+        name: "Dexter",
+        symbol: "DEXTR",
+      },
+      token2: {
+        name: "Radix",
+        symbol: "XRD",
+      },
+    },
+    {
+      name: "ADEX/XRD",
+      token1: {
+        name: "Adex",
+        symbol: "ADEX",
+      },
+      token2: {
+        name: "Radix",
+        symbol: "XRD",
+      },
+    },
+    {
+      name: "CUPPA/XRD",
+      token1: {
+        name: "Cuppa",
+        symbol: "CUPPA",
+      },
+      token2: {
+        name: "Radix",
+        symbol: "XRD",
+      },
+    },
+  ];
+
+  test("should find pairs by full name", () => {
+    const result = searchPairs("DEXTR/XRD", pairsList);
```

Comment:
This means full name of the project, not symbol. So for example XRD is the token of the project Radix. All pairs containing "XRD" should be found when typing "radix" 
```suggestion
    const result = searchPairs("Radix", pairsList);
```

You will need to adapt the expected result as well, as we expect all pairs to be returned (if you use "Radix").